src/fastapibootcamp/handlers/astroplan/models.py

```diff
@@ -0,0 +1,222 @@
+"""Web API models for the astroplan router."""
+
+# In a FastAPI application, you'll have lots of Pydantic models or dataclasses.
+# These are your "models." It's wise to maintain separate models for different
+# areas of your application. In this module we're defining the Pydantic models
+# that define the request and response JSON payloads for the web API. Notice
+# how the response models have class methods that convert domain (internal)
+# models to the web API model. Using this pattern lets you change how data is
+# represented internally or in databases without inadverently changing the
+# public web API.
+
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Self
+
+from astropy.coordinates import SkyCoord
+from fastapi import Request
+from pydantic import BaseModel, Field, field_validator, model_validator
+from safir.datetime import current_datetime
+from safir.pydantic import normalize_isodatetime
+
+from fastapibootcamp.domain.models import Observer, TargetObservability
+
+__all__ = [
+    "ObserverModel",
+    "ObservationRequestModel",
+    "ObservabilityResponseModel",
+]
+
+
+class ObserverModel(BaseModel):
+    """Model for an observer resource response."""
+
+    id: str = Field(
+        ...,
+        description="URL-safe identifier of the site.",
+        examples=["rubin", "rubin-auxtel"],
+    )
+
+    name: str = Field(
+        ...,
+        description="Short name of the observation site.",
+        examples=["Rubin Observatory"],
+    )
+
+    aliases: list[str] = Field(
+        description="Aliases for the observation site.",
+        default_factory=list,
+        examples=[["LSST", "LSST 8.4m", "Rubin"]],
+    )
+
+    local_timezone: str = Field(
+        ...,
+        description="Local timezone of the observation site.",
+        examples=["Chile/Continental"],
```

Comment:
Very minor nit: `Chile/Continental` is a deprecated time zone identifier and not available on many systems (such as the laptop where I'm writing this). It predates the switch to Olson time zone identifiers, which are always of the form Continent/City where the city is the most populous city in the time zone. The current corresponding time zone identifier is `America/Santiago`.

Reply:
Interesting. I'm going by the Astropy data here. I'll see if I can update it and the examples as well.